docs/vlab/running.md

```diff
@@ -29,6 +29,7 @@ ubuntu@docs:~$ hhfab vlab gen
 21:27:16 INF >>> mclagServers=2 eslagServers=2 unbundledServers=1 bundledServers=1
 21:27:16 INF Generated wiring file name=vlab.generated.yaml
 ```
+[Click Here](#build-the-installer-and-start-vlab) to start VLAB, or see the next section for customizing the topology.
```

Comment:
I don't like this, we don't know where the link is taking us. Instead of “Click here” I'd use something like `You can jump straight to [the instructions to start the VLAB](<link>), or see ...`.

Reply:
agreed